userspace/libscap/engine/gvisor/runsc.cpp

```diff
@@ -50,18 +51,14 @@ result runsc(char *argv[])
 			return res;
 		}
 
-		FILE *f = fdopen(pipefds[0], "r");
-		if(!f)
-		{
-			return res;
-		}
+		__gnu_cxx::stdio_filebuf<char> filebuf(pipefds[0], std::ios::in);
```

Comment:
note for the reviewer: this is a GNU extension, but I'm not sure if there is a better way, apart from re-implementing it